docs/src/main/paradox/migration.md

```diff
@@ -23,3 +23,8 @@ jdbc-read-journal {
   dao = "akka.persistence.jdbc.query.dao.legacy.ByteArrayReadJournalDao"
 }
 ```
+
+If you have re-configured the `schemaName`, `tableName` and `columnNames` through configuration settings, you will need to move them a new key.
+
+* key `jdbc-journal.tables.journal` becomes `jdbc-journal.tables.legacy_journal`
+* key `jdbc-journal.tables.snapshot` becomes `jdbc-journal.tables.legacy_snapshot`
```

Comment:
```suggestion
* key `jdbc-snapshot-store.tables.snapshot` becomes `jdbc-snapshot-store.tables.legacy_snapshot`
* key `jdbc-read-journal.tables.journal` becomes `jdbc-read-journal.tables.legacy_journal`
```

Reply:
@octonato here is a proposal to be even more exhaustive with the add of the `jdbc-read-journal` part. Also, I guess the change regarding the snapshot tables is about the `jdbc-snapshot-store` key.

Reply:
You are right, that was a mistake of mine. 

For the read journal is less necessary. The old dao is now reading `jdbc-read-journal.tables.legacy_journal` and the reference.conf takes the value from `jdbc-journal.tables.legacy_journal`, so at the end it will already point to the right place.

Anyway, it does no harm to mention it. Thanks